Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -526,6 +544,33 @@ public void activateNextTasks(List<Task> allHigherTasks) throws DataException, I
         return nextTasks;
     }
 
+    private List<Task> getPreviousTasks(Task higherTask) {
+        List<Task> tasks = higherTask.getProcess().getTasks();
+
+        boolean isConcurrentOpenTask = false;
+        List<Task> previousTasks = new ArrayList<>();
+        List<Task> concurrentTasks = getConcurrentTasksForClose(tasks, higherTask);
+        for (Task concurrentTask : concurrentTasks) {
+            if (concurrentTask.getProcessingStatus().equals(TaskStatus.LOCKED)) {
+                isConcurrentOpenTask = true;
```

Comment:
```suggestion
                isConcurrentOpenTask = true;
                break;
```
Loop can be terminated when condition is met.